web/crux/src/app/node/node.service.ts

```diff
@@ -175,19 +176,79 @@ export default class NodeService {
     return await this.agentService.getNodeEventsByTeam(request.id)
   }
 
-  handleWatchContainerStatus(request: WatchContainerStateRequest): Observable<ContainerStateListMessage> {
+  async handleWatchContainerStatus(
+    request: WatchContainerStateRequest,
+  ): Promise<Observable<ContainerStateListMessage>> {
     this.logger.debug(`Opening container status channel for prefix: ${request.nodeId} - ${request.prefix}`)
```

Comment:
Can you extend the log with the newly added deploymentID?

Reply:
Yes. Sure